packages/fast-components-styles-msft/src/design-system/design-system.schema.ts

```diff
@@ -231,5 +232,23 @@ export default {
             type: "number",
             default: 18,
         },
+        neutralPalette: {
```

Comment:
`neutralPalette` and `accentPalette` are arrays of strings - not strings themselves.

Reply:
`neutralPalette` and `accentPalette` have been omitted